assignment4.Rmd

```diff
@@ -111,8 +120,37 @@ biplot(pca)
 
 #Calculate values for each student that represent these your composite variables and then create a new correlogram showing their relationship to mean_correct.
 
+D5 <- as.matrix(D2) %*% pca$rotation
+
 ```
+
+### Answer
+For PC1, it is highest correlation variation. It is strongly related to problems attempted, mean hint, and mean attempt. Students are trying to solve the problem persistently with reachable resources. 
+
 # Part III  
 
 ## Also in this repository is a data set collected from class that shows how many students thought that a TC program was related to andother TC program. Use PCA to look for components of related programs. Explain why you think there are relationships between these programs.
 
+### Answer
+By sorting the value of PC1, Change leadership, Economics and Education, Education Policy, School Principals, and Social Organizational Psychology are the five programs most correlated to other programs based on what people selected. One explanation is that these programs have similar components covered as other programs. Or, they have broader definition that people consider them as overall related.
```

Comment:
Good interpretation. You can go further to talk about the similarities/features for these groups of programs.